code/modules/mob/mob_helpers.dm

```diff
@@ -221,9 +221,8 @@
 			if(1,3,5,8)	newletter="[lowertext(newletter)]"
 			if(2,4,6,15)	newletter="[uppertext(newletter)]"
 			if(7)	newletter+=pick(slurletters)
```

Comment:
Ah yes why use logical X to Y when you can put random numbers in here.
Either somebody was slurring themselves when they wrote this. Or they are trying to counteract the bias of byonds random number generator.

Reply:
Yeah, god this is so archaic. I don't know much about the bias of byond's RNG system though.